### ISSUE TITLE
right inlet of [* ] doesn't work in PD

### ISSUE THREAD

**Issue by jukkapoika** [enzienaudio/heavy#218]
Hi,
It seems like that right inlet of multiplication object doesn't work when compiling pd patches with Heavy. Same goes at least with [/ ]. I am compiling the patch to Bela.

**Comment by mhroth** [comment]
What do you mean exactly, "doesn't work"?

**Comment by jukkapoika** [comment]
I can't seem to set the argument values through right inlet to the object.

for example:

|             [ 3]
|       /
[ * 1]

to change the multiplication from 1 to 3.

**Comment by jukkapoika** [comment]
Actually I found the answer from some other patches on this forum:


|    [1)
|    [ t  b  f]
| / |
[pack]
|
[ €1 €2)
|
[* ]

**Comment by diplojocus** [comment]
@jukkapoika this seems really strange, would you be able to upload a simple patch here that demonstrates your problem?

**Comment by jukkapoika** [comment]
Sorry I have missed this comment somehow. Sure I'll do that hopefully tomorrow!

**Comment by jukkapoika** [comment]
Yep. Master works also for me. So it was just my mistake somewhere.

**Comment by diplojocus** [comment]
Hey @jukkapoika I saw a comment here from you but it's disappeared, did you delete it?

**Comment by jukkapoika** [comment]
Yes. I was accidentally writing a comment here, when I was supposed to write it to another forum. Sorry about that.

**Comment by jukkapoika** [comment]
Here are two examples. So the first one is work around and works when the second one doesn't.

[peak_track_patches.zip](https://github.com/enzienaudio/heavy/files/1116442/peak_track_patches.zip)
